packages/cli-kit/src/git.ts

```diff
@@ -12,12 +13,61 @@ export const GitNotPresentError = () => {
   )
 }
 
+export const OutsideGitDirectoryError = () => {
+  return new Abort('Must be inside a Git directory to continue')
+}
+
+export const NoCommitError = () => {
+  return new Abort(
+    'Must have at least one commit to run command',
```

Comment:
Is this Git helper generic? Hence just `run command`? If it's specific to the deploy command, then let's use the actual command:

```suggestion
    'Git repo needs at least one commit to deploy.',
```

Reply:
@gfscott it's in CLI kit so should stay generic. The caller may, of course, raise a new error if desired.